lms/djangoapps/program_enrollments/constants.py

```diff
@@ -0,0 +1,33 @@
+"""
+Constants used throughout the program_enrollments app and exposed to other
+in-process apps through api.py.
+"""
+
+
+class ProgramEnrollmentStatuses(object):
+    """
+    Status that a user may have enrolled in a program.
+
+    Note: Changing the value of __ALL__ will trigger a migration.
+
+    TODO: Define the semantics of each of these (EDUCATOR-4958)
+    """
+    ENROLLED = 'enrolled'
+    PENDING = 'pending'
+    SUSPENDED = 'suspended'
+    CANCELED = 'canceled'
+    __ACTIVE__ = (ENROLLED, PENDING)
+    __ALL__ = (ENROLLED, PENDING, SUSPENDED, CANCELED)
```

Comment:
Nit: I like how you made it clear that this variable has other dependencies, maybe consider a less generic name for it?  Perhaps `__MODEL_CHOICES__`?

Reply:
I think I like the more generic name because there are non-model contexts where `__ALL__` is used (e.g., serializers). But explicit naming is also good.

How's this?
```python
...
__ALL__ = (ENROLLED, PENDING, SUSPENDED, CANCELED)

# Note: Any changes to this value will trigger a migration on ProgramEnrollment
__MODEL_CHOICES__ = __ALL__